NedyUdombat/ReachHigh
# Setup postgreSQL and sequelize

#### What does this PR do?
Setup postgreSQL and sequelize

#### Description of Task to be completed?
- [x] Install relevant dependencies
- [x] Setup sequelize configuration and connection
- [x] Setup migration and models

#### How should this be manually tested?
* clone this repo
* checkout to `ch-setup-postgres-sequelize-169169087 ` branch
* run `npm install`
* run `npm install -g sequelize-cli` to install the sequelize cli globally
* run `cp .env.example .env`
* create a local postgres database and fill in the required details in the env
* run `sequleize db:migrate`
* Check your database you will find the created tables

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?(if applicable)
[Pivotal tracker ID](https://www.pivotaltracker.com/story/show/169169087/)

#### Screenshots
